main.css

```diff
@@ -47,6 +47,21 @@ body {
 	word-spacing: 0.1rem;
 }
 
+::-webkit-scrollbar {
+    width: 12px;
+}
+
+::-webkit-scrollbar-thumb {
+    background: linear-gradient(45deg, #af08f7, #a407b0, #4a0259);
+    border-radius: 10px;
+	border: 1px solid #1A1C29;
+}
+
+::-webkit-scrollbar-track {
+    background: #222121;
+    border-radius: 10px;
+}
```

Comment:
let's try removing this
```suggestion
```

Reply:
Sir i havent changed anything yet
i am just letting you know.
@yurijmikhalevich